### PR TITLE
Run3-sim80 Make the Cherenkov example working for DDD

### PR DESCRIPTION
**Pull request by bsunanda**
#### PR description:

Make the Cherenkov example working for DDD

#### PR validation:

Tested with the example cfg in SimG4CMS/CherenkovAnalysis/test

#### if this PR is a backport please specify the original PR and why you need to backport that PR:

Nothing special

### CODE REVIEW

**Comment by bsunanda** [comment]
@cmsbuild Please test

File: Geometry/HcalCommonData/src/CaloSimParametersFromDD.cc [hunk 1]
```diff
@@ -8,7 +8,7 @@
 #include <iostream>
 #include <iomanip>
 
-//#define EDM_ML_DEBUG
+#define EDM_ML_DEBUG
```
Comment (civanch) [inline Geometry/HcalCommonData/src/CaloSimParametersFromDD.cc:11]: @bsunanda , likely this should be commented out

Comment (bsunanda) [reply Geometry/HcalCommonData/src/CaloSimParametersFromDD.cc]: Taken care of

**Comment by bsunanda** [comment]
@cmsbuild Please test

**Comment by civanch** [comment]
+1

**Comment by qliphy** [comment]
+1

**Pull request merged**